test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Wide.Tests.ps1

```diff
@@ -1,33 +1,39 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Describe "Format-Wide" -Tags "CI" {
+    BeforeAll {
+        1..2 | ForEach-Object { New-Item -Path ("TestDrive:\Testdir{0:00}" -f $_) -ItemType Directory }
+        1..2 | ForEach-Object { New-Item -Path ("TestDrive:\TestFile{0:00}.txt" -f $_) -ItemType File }
+        $pathList = Get-ChildItem $TestDrive
+    }
 
     It "Should have the same output between the alias and the unaliased function" {
-        $nonaliased = Get-ChildItem | Format-Wide
-        $aliased    = Get-ChildItem | fw
+        $nonaliased = $pathList | Format-Wide
+        $aliased = $pathList | fw
```

Comment:
Please replace the alias.

Reply:
What does `replace` mean?
This test compares the results of alias and unaliased command.

Reply:
Ah, sorry. We have special tests for aliases and shouldn't test it here. Pleas remove.

Reply:
I understand. I will remove this test.